clients/js/main.ts

```diff
@@ -22,48 +22,14 @@ import yargs from "yargs";
 import { hideBin } from "yargs/helpers";
 
 import { fromBech32, toHex } from "@cosmjs/encoding";
-import { CONTRACTS as SDK_CONTRACTS } from "@certusone/wormhole-sdk";
-import {
-  isTerraChain,
-  assertEVMChain,
-  setDefaultWasm,
-  hexToUint8Array,
-  getEmitterAddressSolana,
-  getEmitterAddressTerra,
-  getEmitterAddressEth,
-  getEmitterAddressAlgorand,
-  getEmitterAddressNear,
-  isCosmWasmChain,
-} from "@certusone/wormhole-sdk";
-import { execute_solana } from "./solana";
-import {
-  execute_evm,
-  getImplementation,
-  hijack_evm,
-  query_contract_evm,
-  setStorageAt,
-} from "./evm";
-import { execute_terra } from "./terra";
-import { execute_aptos } from "./aptos";
-import { execute_near, upgrade_near, deploy_near } from "./near";
 import * as vaa from "./vaa";
 import { impossible, Payload, serialiseVAA, VAA } from "./vaa";
-import {
-  assertChain,
-  ChainName,
-  CHAINS,
-  toChainName,
-  isEVMChain,
-  toChainId,
-  tryNativeToHexString,
-} from "@certusone/wormhole-sdk";
 import { ethers } from "ethers";
 import { NETWORKS } from "./networks";
 import base58 from "bs58";
-import { execute_algorand } from "./algorand";
-import { execute_injective } from "./injective";
-import { execute_xpla } from "./xpla";
 import { isOutdated } from "./cmds/update";
+import { setDefaultWasm } from "@certusone/wormhole-sdk/lib/cjs/solana/wasm";
+import { assertChain, assertEVMChain, ChainName, CHAINS, CONTRACTS as SDK_CONTRACTS, isCosmWasmChain, isEVMChain, isTerraChain, toChainId, toChainName } from "@certusone/wormhole-sdk/lib/cjs/utils/consts";
```

Comment:
Prettier will probably change this line the next time someone saves this file.  Can you run prettier on this file?

Reply:
ah drat, I missed this. since we have the approvals now, I'd rather merge it in and sort out prettier stuff when it's mandatory in CI

Reply:
no, turns out I didn't miss it, it's just that the auto-formatter I have enabled by default (`tsfmt`) keeps it this way.